src/screens/CreatePost/index.js

```diff
@@ -52,6 +53,7 @@ import useCreatePostHook from '../../hooks/screen/useCreatePostHook';
 import useHastagMention from './elements/useHastagMention';
 import {Analytics} from '../../libraries/analytics/firebaseAnalytics';
 import {Button, ButtonAddMedia} from '../../components/Button';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import for `COLORS` from '../../utils/theme' has been moved to line 56. This change seems fine as long as it doesn't affect the order of execution or cause any hoisting issues.
```diff
+ import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->